packages/catana/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=catana
+pkgver=1.0
+pkgrel=1
+pkgdesc='CATANA filters your wordlist according to the specified password policy.'
```

Comment:
Can you remove tool name from the description? it's redundant

Reply:
Deleting application name from description.

Reply:
Commit: 49865587c417d0345a817706d2cab8f25cc7506f